gpt_index/vector_stores/redis.py

```diff
@@ -0,0 +1,313 @@
+"""Redis Vector store index.
+
+An index that that is built on top of an existing vector store.
+"""
```

Comment:
nit: this should just be vector store now, no need to mention index anymore after refactor.